startup/99-helper_functions.py

```diff
@@ -59,7 +69,7 @@ def pol_L(pol, epu_cal_offset=None, epu_table_number=None):
     
     
     yield from bps.mv(diag6_pid.enable,1) # finepitch feedback ON
-    yield from bps.sleep(1)
+    yield from bps.sleep(0.1)
```

Comment:
Sleeps frighten me, although less so to see an existing one reduced, is there something this is waiting on?

Reply:
we can reduce the time because we are able to run the diag6 prosilica much faster.  diag6_pid is the pid loop that keeps the m1 finepitch under constant control to keep the beam on the exit slit.  There could be some instability in the beginning of the feedback loop when we turn it back on.  The sleep is to allow that instability to happen without letting the user collect data.  the daig6_pid knows nothing about the output error that is running at dt = 0.005 seconds or faster.